src/main/java/org/traccar/config/Keys.java

```diff
@@ -1198,6 +1198,13 @@ private Keys() {
             "web.sameSiteCookie",
             Collections.singletonList(KeyType.GLOBAL));
 
+    /**
+     * Enables persisting Jetty session to the database
+     */
+    public static final ConfigKey<Boolean> WEB_SESSION_PERSISTED = new ConfigKey<>(
```

Comment:
One other minor thing. Can we please rename it to be "persisted session" or "persist session" instead of "session persisted".